src/piracy.jl

```diff
@@ -105,7 +105,7 @@ end
 is_foreign(@nospecialize(T::TypeVar), pkg::Base.PkgId) = is_foreign(T.ub, pkg)
 
 # Before 1.7, Vararg was a UnionAll, so the UnionAll method will work
-@static if VERSION >= v"1.7"
+@static if VERSION >= v"1.7-"
```

Comment:
Huh, what is the difference in behavior herr?

Reply:
the new comparison is also true for all beta's and rc's before the 1.7 release. As the behavior tested here was changed before the 1.7 feature freeze, this should be changed, as it currently breaks tests on e.g. `1.7.0-beta1` and `1.7.0-rc3`. Changing this fixes it.

Reply:
OK, so this is for a hypothetical scenario were someone is *today* using a beta version of an outdated non-LTS Julia version... got it ;-)